MergeSortedArray.java

```diff
@@ -0,0 +1,37 @@
+// Time Complexity : O(n)
+// Space Complexity : O(1)
```

Comment:
Time complexity should be O(m+n)::m and n be the number of first and second array respectively